Missing apollo-upload-server dependency

**Intended outcome:**

`apollo-server-lambda@rc` can be used.

**Actual outcome:**

Failure due to missing `apollo-upload-server` dependency. Actual error is: `"Cannot find module 'apollo-upload-server'"`. **I see that `apollo-upload-server` is referenced by `apollo-server-core` [here](https://github.com/apollographql/apollo-server/blob/742fb41bc37294717cd0a9a52f2cf5658abdba7e/packages/apollo-server-core/src/ApolloServer.ts#L17), but it's not included in the core's [dependencies](https://github.com/apollographql/apollo-server/blob/742fb41bc37294717cd0a9a52f2cf5658abdba7e/packages/apollo-server-core/package.json#L51-L69). Could that be the issue?** 

Adding `apollo-upload-server` to the dependencies of `apollo-server-core` should fix this issue. Let me know if you agree, and I'll be happy to submit a PR.

**More info:**

* Node version: v8.10.
* Using `serverless` framework.
* `graphql`'s lambda handler:

```js
const { ApolloServer } = require('apollo-server-lambda');

const resolvers = require('./data/resolvers');
const typeDefs = require('./data/typeDefs');

const server = new ApolloServer({
  typeDefs,
  resolvers,
});

exports.handler = server.createHandler();
```

--- a/packages/apollo-server-core/package.json
+++ b/packages/apollo-server-core/package.json
@@ -27,6 +27,26 @@
   "engines": {
     "node": ">=6"
   },
+  "dependencies": {
+    "@types/ws": "^5.1.2",
+    "apollo-cache-control": "^0.1.1",
+    "apollo-datasource-rest": "^2.0.0-rc.1",
+    "apollo-engine-reporting": "0.0.0-beta.14",
+    "apollo-server-env": "^2.0.0-rc.1",
+    "apollo-server-errors": "^2.0.0-rc.1",
+    "apollo-tracing": "^0.2.0-beta.1",
+    "apollo-upload-server": "^5.0.0",
+    "graphql-extensions": "0.1.0-beta.13",
+    "graphql-subscriptions": "^0.5.8",
+    "graphql-tag": "^2.9.2",
+    "graphql-tools": "^3.0.2",
+    "hash.js": "^1.1.3",
+    "keyv": "^3.0.0",
+    "lodash": "^4.17.10",
+    "quick-lru": "^1.1.0",
+    "subscriptions-transport-ws": "^0.9.9",
+    "ws": "^5.2.0"
+  },
   "devDependencies": {
     "@types/fibers": "0.0.30",
     "@types/graphql": "^0.13.1",
@@ -47,25 +67,5 @@
   "typings": "dist/index.d.ts",
   "typescript": {
     "definition": "dist/index.d.ts"
-  },
-  "dependencies": {
-    "@types/ws": "^5.1.2",
-    "apollo-cache-control": "^0.1.1",
-    "apollo-datasource-rest": "^2.0.0-rc.1",
-    "apollo-engine-reporting": "0.0.0-beta.14",
-    "apollo-server-env": "^2.0.0-rc.1",
-    "apollo-server-errors": "^2.0.0-rc.1",
-    "apollo-tracing": "^0.2.0-beta.1",
-    "apollo-upload-server": "^5.0.0",
-    "graphql-extensions": "0.1.0-beta.13",
-    "graphql-subscriptions": "^0.5.8",
-    "graphql-tag": "^2.9.2",
-    "graphql-tools": "^3.0.2",
-    "hash.js": "^1.1.3",
-    "keyv": "^3.0.0",
-    "lodash": "^4.17.10",
-    "quick-lru": "^1.1.0",
-    "subscriptions-transport-ws": "^0.9.9",
-    "ws": "^5.2.0"
   }
 }

--- a/packages/apollo-server-core/package.json
+++ b/packages/apollo-server-core/package.json
@@ -56,6 +56,7 @@
     "apollo-server-env": "^2.0.0-rc.1",
     "apollo-server-errors": "^2.0.0-rc.1",
     "apollo-tracing": "^0.2.0-beta.1",
+    "apollo-upload-server": "^5.0.0",
     "graphql-extensions": "0.1.0-beta.13",
     "graphql-subscriptions": "^0.5.8",
     "graphql-tag": "^2.9.2",

